frontend/src/css/modules/my-challenge-all-submission.scss

```diff
@@ -7,3 +7,15 @@
 .all-submission-table-scroll {
 	overflow-x: scroll;
 }
+
+.all-submission-card {
```

Comment:
Reuse the classes for this card from other elements. No need to add a new class.